src/System.CommandLine.Tests/DirectiveTests.cs

```diff
@@ -57,7 +58,61 @@ public void Directives_must_be_the_first_argument()
 
             var result = option.Parse("-y [suggest]");
 
-            result.UnmatchedTokens.Should().Contain("[suggest]"); 
+            result.UnmatchedTokens.Should().Contain("[suggest]");
+        }
+
+        [Theory]
+        [InlineData("key:value", "key", "value")]
+        [InlineData("key:value:more", "key", "value:more")]
+        [InlineData("key:", "key", "")]
```

Comment:
Minor thing, but I think it would be easier to follow if we left the braces as part of the inline data. Then it would match the other unit tests in this file as well.